src/index.html

```diff
@@ -1,13 +1,64 @@
 <!doctype html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect"
+          href="https://fonts.googleapis.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap"
+          rel="stylesheet">
+    <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Product cards</h1>
+  <body class="page__body">
+    <div class="card" data-qa="card">
+      <div class="card__img-wrapper">
+        <img
+          class="card__img"
+          src="./images/imac.jpeg"
+          alt="imac"
+        >
+      </div>
+
+      <div class="card__content">
+        <h4 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock
+          (MNED2UA/A)
+        </h4>
+
+        <p class="card__code">Product code: 195434</p>
+
+        <div class="card__review">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+
+            <div class="stars__star"></div>
+
+            <div class="stars__star"></div>
+
+            <div class="stars__star"></div>
+
+            <div class="stars__star"></div>
+          </div>
+
+          <p class="card__reviews-number">Reviews: 5</p>
+        </div>
```

Comment:
```suggestion
        <div class="card__review">
          <div class="stars stars--4">
            <div class="stars__star"></div>
            <div class="stars__star"></div>
            <div class="stars__star"></div>
            <div class="stars__star"></div>
            <div class="stars__star"></div>
          </div>

          <p class="card__reviews-number">Reviews: 5</p>
        </div>
```
there is no need to add spaces between one line sibling elements